### ISSUE TITLE
General Instruction to access webmail 

### ISSUE THREAD

**Issue by ayan-biswas0412** [ayan-biswas0412/Juetcecoursemanagementsystem#65]
Please follow this and link your personal email account with the Roundcube email client (click on the webmail at homepage ).

See this for first time installation : https://youtu.be/wgbf9N3wpaA

**Comment by ayan-biswas0412** [comment]
But don't remove the installer folder beacuse it have to be done when we will install it finally at lab computer for testing purpose keeo the folder always

**Comment by ayan-biswas0412** [comment]
Pull it from 'ayan/develop' branch

**Comment by arindammajee** [comment]
This article is also helpful. Here the command lines are given one by one.
https://websiteforstudents.com/install-roundcube-webmail-on-ubuntu-16-04-18-04-with-apache2-and-php-7-2-support/

**Comment by ayan-biswas0412** [comment]
> This article is also helpful. Here the command lines are given one by one.
> https://websiteforstudents.com/install-roundcube-webmail-on-ubuntu-16-04-18-04-with-apache2-and-php-7-2-support/

Yes it is also helpful and also we have to mention in the documentation that how a user will access the webmail (for an end-user like faculty/student /admin ) they will not install this type of stuffs they will just link IMAP or POP3 support via their email and they can then easily login to this email .

This  thing is needed to be mentioned in document
